docs/docsite/rst/conf.py

```diff
@@ -386,3 +392,22 @@
 ]
 linkcheck_workers = 25
 # linkcheck_anchors = False
+
+# Generate redirects for pages when building on Read The Docs
+def setup(app: Sphinx) -> dict[str, str]:
```

Comment:
look at the return statement, most vals are bools
```suggestion
def setup(app: Sphinx) -> dict[str, bool | str]:
```